controller/graphql/server.go

```diff
@@ -94,11 +131,26 @@ func GetHandler(db state.State, accessToken string) (*http.ServeMux, error) {
 		return nil, err
 	}
 
+	loader := func(ctx context.Context, cl cidlink.Link, builder ipld.NodeBuilder) (ipld.Node, error) {
+		store := db.Store(ctx)
+		block, err := store.Get(cl.Cid)
+		if err != nil {
+			return nil, err
+		}
+		if err := dagjson.Decoder(builder, bytes.NewBuffer(block.RawData())); err != nil {
+			return nil, err
+		}
+
+		n := builder.Build()
+		return n, nil
+	}
+
 	mux := http.NewServeMux()
 	mux.Handle("/", http.FileServer(http.FS(index)))
 	mux.Handle("/graphql", CorsMiddleware(func(w http.ResponseWriter, r *http.Request) {
 		var result *graphql.Result
-		ctx := r.Context()
+		sctx := r.Context()
```

Comment:
I guess you don't need this variable? having fewer names in scope makes it easier for me to read the code at least, since I need to worry about fewer names at any point.